docs/TESTS.md

```diff
@@ -1,15 +1,157 @@
 # Tests
 
-<!-- TODO: write document
+This track uses the [unitt unit-testing framework][unitt] which should be installed automatically when you run the tests.
+Some tests are skipped by default when working offline, but all tests are run by the online test runner.
 
-  This document should describe everything related to running tests in the track.
+```
+.../arturo/hello-world
+HELP.md
+README.md
+src
+tester.art
+tests
+```
 
-  If your track uses skipped tests, this document can explain why thet is used and
-  how to unskip tests.
+Each exercise folder contains a `tester.art` entry point (see below).
```

Comment:
The files are listed above this line. Do you want to move those lines or change this one?